docs/docs/en/guide/start/docker.md

```diff
@@ -72,7 +72,7 @@ $ DOLPHINSCHEDULER_VERSION=<version>
 # Initialize the database, make sure database <DATABASE> already exists
 $ docker run -d --name dolphinscheduler-tools \
     -e DATABASE="postgresql" \
-    -e SPRING_DATASOURCE_URL="jdbc:postgresql://localhost:5432/<DATABASE>" \
+    -e SPRING_DATASOURCE_URL="jdbc:postgresql://localhost:5432/dolphinscheduler" \
```

Comment:
I don't think this is a typo since we already have comment above.

Reply:
So I think maybe we should correct below 4 into DATABASE?
It is a little bit confuse with SPRING_DATABASE configuration.

Reply:
change others components env `SPRING_DATASOURCE_URL` LGTM

Reply:
> change others components env `SPRING_DATASOURCE_URL` LGTM

Thanks for discussion.
Updated.